conference.js

```diff
@@ -2184,10 +2186,26 @@ export default {
                     return stream;
                 })
                 .then(stream => {
-                    this.useVideoStream(stream);
+                    // if we are screen sharing we do not want to stop it
+                    if (this.isSharingScreen) {
+                        return Promise.resolve();
+                    }
+
+                    return this.useVideoStream(stream);
+                })
+                .then(() => {
                     logger.log('switched local video device');
-                    APP.settings.setCameraDeviceId(cameraDeviceId, true);
+
+                    // If video was muted before changing current device
+                    // and selected new one, then mute new video track.
+                    if (!this.isSharingScreen
+                        && videoWasMuted) {
+                        sendEvent('changeDevice.video.muted');
+                        muteLocalVideo(true);
```

Comment:
Is there a flash of video visible using a video stream and then muting?

Reply:
I don't see that, and I was checking that on other places we do this first useVideoStream and then muteLocalVideo.

Reply:
I tested locally and I was able to see a flash of video on a 1:1 call. Is it easy enough to mute the track and then use it?